src/modules/users/crons/application-status-update.cron.ts

```diff
@@ -0,0 +1,118 @@
+import { UserApplication } from '@entities/user_applications.entity';
+import { Injectable, Logger } from '@nestjs/common';
+import { Cron } from '@nestjs/schedule';
+import { InjectRepository } from '@nestjs/typeorm';
+import { In, Not, Repository } from 'typeorm';
+import { v4 as uuidv4 } from 'uuid';
+import { ConfigService } from '@nestjs/config';
+import { ProxyService } from '@services/proxy/proxy.service';
+
+@Injectable()
+export class ApplicationStatusUpdate {
+  constructor(
+    @InjectRepository(UserApplication)
+    private readonly userApplicationRepository: Repository<UserApplication>,
+    private readonly configService: ConfigService,
+    private readonly proxyService: ProxyService,
+  ) {}
+
+  async getApplications() {
+    try {
+      const applications = await this.userApplicationRepository.find({
+        where: {
+          status: Not(In(['amount received', 'rejected'])),
```

Comment:
do we have list of all statuses? `disbursed` one also should be ignored?

Reply:
are statuses coming in lowecase?

Reply:
@Vidyaranya-Gavai-TTPL Please check the status for their spelling and underscores i mean is the status as 
amount_received etc and also are we supposed to check only 2 status?